app/src/main/java/de/westnordost/streetcomplete/map/MainFragment.kt

```diff
@@ -607,6 +611,25 @@ class MainFragment : Fragment(R.layout.fragment_main),
         mapFragment?.updateCameraPosition(300) { zoomBy = +1f }
     }
 
+    private fun onClickTracksStop() {
+
+        // hide the track information
```

Comment:
This is kind of pedantic, but elsewhere in the app, we don't have empty lines at the beginning and end of functions.
```suggestion
    private fun onClickTracksStop() {
        // hide the track information
```